pogobuf/pogobuf.utils.js

```diff
@@ -248,6 +248,30 @@ module.exports = {
     },
 
     /**
+     * Utility method to get the distance in meters between two locations
+     * @param {number} lat1 - origin latitude
+     * @param {number} lng1 - origin longitude
+     * @param {number} lat2 - destination latitude
+     * @param {number} lng2 - destination longitude
```

Comment:
They could be, but origin/destination are not really a part of the formula. It just calculates distance between two points, which could technically be flipped to destination/origin with the same result.